bot/cogs/watchchannels/watchchannel.py

```diff
@@ -0,0 +1,353 @@
+
+                    self.log.trace(f"Consuming message {msg.id} ({len(msg.attachments)} attachments)")
+                    await self.relay_message(msg)
+
+        self.consumption_queue.clear()
+
+        if self.message_queue:
+            self.log.trace("Channel queue not empty: Continuing consuming queues")
+            self._consume_task = self.bot.loop.create_task(self.consume_messages(delay_consumption=False))
+        else:
+            self.log.trace("Done consuming messages.")
+
+    async def webhook_send(
+        self,
+        content: Optional[str] = None,
+        username: Optional[str] = None,
+        avatar_url: Optional[str] = None,
+        embed: Optional[Embed] = None,
+    ) -> None:
+        """Sends a message to the webhook with the specified kwargs."""
+        try:
+            await self.webhook.send(content=content, username=username, avatar_url=avatar_url, embed=embed)
+        except discord.HTTPException as exc:
+            self.log.exception(
+                f"Failed to send message to {self.__class__.__name__} webhook",
+                exc_info=exc
+            )
+
+    async def relay_message(self, msg: Message) -> None:
+        """Relays the message to the relevant watch channel"""
+        last_author, last_channel, count = self.message_history
+        limit = BigBrotherConfig.header_message_limit
+
+        if msg.author.id != last_author or msg.channel.id != last_channel or count >= limit:
+            self.message_history = MessageHistory(last_author=msg.author.id, last_channel=msg.channel.id)
+
+            await self.send_header(msg)
+
+        cleaned_content = msg.clean_content
+
+        if cleaned_content:
+            # Put all non-media URLs in a code block to prevent embeds
+            media_urls = {embed.url for embed in msg.embeds if embed.type in ("image", "video")}
+            for url in URL_RE.findall(cleaned_content):
+                if url not in media_urls:
+                    cleaned_content = cleaned_content.replace(url, f"`{url}`")
+            await self.webhook_send(
+                cleaned_content,
+                username=msg.author.display_name,
+                avatar_url=msg.author.avatar_url
+            )
+
+        if msg.attachments:
+            try:
+                await messages.send_attachments(msg, self.webhook)
+            except (errors.Forbidden, errors.NotFound):
+                e = Embed(
+                    description=":x: **This message contained an attachment, but it could not be retrieved**",
+                    color=Color.red()
+                )
+                await self.webhook_send(
+                    embed=e,
+                    username=msg.author.display_name,
+                    avatar_url=msg.author.avatar_url
+                )
+            except discord.HTTPException as exc:
+                self.log.exception(
+                    f"Failed to send an attachment to {self.__class__.__name__} webhook",
```

Comment:
Remove the name from the message.

```suggestion
                    f"Failed to send an attachment to the webhook",
```